apps/web/src/pages/EmployeeProfile/components/WfaSection/Display.tsx

```diff
@@ -0,0 +1,120 @@
+import { useIntl } from "react-intl";
+
+import {
+  EmployeeProfileWfaFragment,
+  WfaInterest,
+} from "@gc-digital-talent/graphql";
+import {
+  commonMessages,
+  getWfaInterestFirstPerson,
+} from "@gc-digital-talent/i18n";
+import { unpackMaybes } from "@gc-digital-talent/helpers";
+import { Ul } from "@gc-digital-talent/ui";
+
+import ToggleForm from "~/components/ToggleForm/ToggleForm";
+import ExperienceCard from "~/components/ExperienceCard/ExperienceCard";
+import { formattedDate } from "~/utils/dateUtils";
+import processMessages from "~/messages/processMessages";
+
+import messages from "../../messages";
+
+interface DisplayProps {
+  user: EmployeeProfileWfaFragment;
+}
+
+const Display = ({ user }: DisplayProps) => {
+  const intl = useIntl();
+  const notProvided = intl.formatMessage(
+    commonMessages.missingOptionalInformation,
+  );
+  const experiences = unpackMaybes(user.currentSubstantiveExperiences);
+  const communities = unpackMaybes(user.employeeProfile?.communityInterests);
+
+  return (
+    <div className="flex flex-col gap-y-6">
+      <ToggleForm.FieldDisplay
+        label={intl.formatMessage(messages.wfaSituation)}
+      >
+        {user.employeeWFA?.wfaInterest?.value
+          ? intl.formatMessage(
+              getWfaInterestFirstPerson(user.employeeWFA.wfaInterest.value),
+            )
+          : notProvided}
+      </ToggleForm.FieldDisplay>
+      {user.employeeWFA?.wfaInterest?.value &&
+        user.employeeWFA.wfaInterest.value !== WfaInterest.NotApplicable && (
+          <>
+            <ToggleForm.FieldDisplay
+              label={intl.formatMessage(messages.currentSubstantivePosition)}
+            >
+              {experiences.length > 0 ? (
+                <div className="mt-3 flex flex-col gap-y-3">
+                  {experiences.map((exp) => (
+                    <ExperienceCard
+                      key={exp.id}
+                      experienceQuery={exp}
+                      showEdit={false}
+                    />
+                  ))}
+                </div>
+              ) : (
+                notProvided
+              )}
+            </ToggleForm.FieldDisplay>
+            <ToggleForm.FieldDisplay
+              label={intl.formatMessage(messages.expectedEndDate)}
+            >
+              {user.employeeWFA.wfaDate
+                ? formattedDate(user.employeeWFA.wfaDate, intl)
+                : notProvided}
+            </ToggleForm.FieldDisplay>
+            <ToggleForm.FieldDisplay
+              label={intl.formatMessage(messages.currentCommunity)}
+            >
+              {communities.length > 0 ? (
+                <Ul space="sm">
+                  {communities.map((interest) => (
+                    <li key={interest.community.id}>
+                      {interest?.community?.name?.localized ??
+                        intl.formatMessage(commonMessages.notAvailable)}
+                    </li>
```

Comment:
Empty functional community state should be like an error

Figma

<img width="560" height="478" alt="image" src="https://github.com/user-attachments/assets/2dfad1a4-8fb5-48d6-9968-dec87c8ff756" />

Current

<img width="770" height="139" alt="image" src="https://github.com/user-attachments/assets/76d959fa-dfd9-4cb0-b1ab-0badffe1bd9b" />